fdbserver/ptxn/TLogServer.actor.cpp

```diff
@@ -887,7 +886,7 @@ void commitMessages(Reference<TLogGroupData> self,
 	StringRef storedMessage(block.end() - msgSize, msgSize);
 	const auto expectedStoredMessageSize = storedMessage.expectedSize();
 
-	storageTeamData->versionMessages[version] = std::make_pair(storedMessage, block.arena());
+	storageTeamData->versionMessages[version] = Standalone(storedMessage); // do memcpy to copy contents from block
```

Comment:
We can avoid `memcpy` by:
`storageTeamData->versionMessages[version] = Standalone(storedMessage, block.arena());`